dash-pipeline/dpapp/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
do you mind to add a README here to help people getting started on it?

Reply:
done